ui/app/pages/swaps/dropdown-input-pair/dropdown-input-pair.js

```diff
@@ -92,6 +93,7 @@ export default function DropdownInputPair ({
           fullWidth
           margin="dense"
           value={ inputValue }
+          {...(autoFocus && { autoFocus: true })}
```

Comment:
Interesting :thinking:  Any reason you couldn't set `autoFocus` if it was falsey?

Reply:
TIL `autoFocus` truly is a bool.  Updated!